doc-content/enterprise-only/vehicle-route-planning/vrp-troubleshooting-proc.adoc

```diff
@@ -8,9 +8,9 @@ If the OptaWeb Vehicle Routing behaves unexpectedly, follow this procedure to tr
 * OptaWeb Vehicle Routing is running and behaving unexpectedly.
 
 .Procedure
-. To identify issues, review the back end terminal output log.
-. To resolve issues, remove the back end database:
+. To identify issues, review the back-end terminal output log.
+. To resolve issues, remove the back-end database:
 
-.. Stop the back end by pressing kbd:[Ctrl+C] in the back end terminal window.
+.. Stop the back-end by pressing kbd:[Ctrl+C] in the back-end terminal window.
```

Comment:
"Stop the back end by..." I'd say the first occurrence is not an adjective.

Reply:
Right!